src/main.cpp

```diff
@@ -25,44 +26,61 @@
 #include "brewtarget.h"
 #include "database.h"
 
-void importFromXml(const QString & optionValue);
-void createBlankDb(const QString & optionValue);
+void importFromXml(const QString & filename);
+void createBlankDb(const QString & filename);
 
 int main(int argc, char **argv)
 {  
    QApplication app(argc, argv);
    app.setOrganizationName("brewtarget");
+#ifdef QT_DEBUG
+   app.setApplicationName("brewtarget-debug");
```

Comment:
This is intended to use a separate set of options when built in debug mode, right? If so, can you put a comment to that effect here?